tests/e2e/fixtures/statsFixtures.js

```diff
@@ -18,6 +18,11 @@ const seasonFixtures = [
     startTime: dayjs('2022-02-16 18:00').valueOf(),
     endTime: dayjs('2022-03-16 18:00').valueOf(),
   },
+  {
+    name: 'Hearts 2023',
+    startTime: dayjs('2023-07-06 18:00').valueOf(),
+    endTime: dayjs('2023-10-05 18:00').valueOf(),
```

Comment:
The problem with this setup is that the point of this season fixture is to test things for the current season, but this will be in the past after October 5th. The better way to do it is to set the dates at runtime based on the current time, in a similar way that the "Future Spades Season" and "World Championship Season" are constructed.

The issue with that is that the "World Championship Season" is the one that is written to be "Current" because it starts one second in the past and ends 13 weeks from now. The World Championships in the current architecture are actually intended to be ~1 second long with no games played or matches associated with them because they represent a tournament and not regular match play.

So: I think we should rename the fixture that is currently called "World Championship Season" to be called "Current Season", then add a match and two games that go with it to the fixtures below. Then we can rename your "Hearts 2023" fixture to "World Championship" and set its start time to 10 seconds in the past (see how this is done in the ones below, using `dayjs().subtract()`), and so its end time is 9 seconds in the past i.e. it started ten seconds ago and lasted 1 second.

We will still likely need to update some of the tests in stats.spec so they accurately reflect that the "Current Season" appears first in the list and the "World Championship" is next after that